### PR TITLE
🏃‍♀️Use stretchr

### PR DESCRIPTION
**Pull request by ashish-amarnath**
Signed-off-by: Ashish Amarnath <ashisham@vmware.com>

### CODE REVIEW

File: velero-csi-plugin/csi_restorer_test.go [hunk 1]
```diff
@@ -96,26 +98,21 @@ func TestResetPVCAnnotations(t *testing.T) {
...
-			if tc.pvc.Annotations == nil {
-				t.Errorf("%s failed, unexpected PVC annotations returned, Want: non-nil; Got: nil", tc.name)
-			}
-			if tc.expectNonEmptyAnnotations && len(tc.pvc.Annotations) != 0 {
-				t.Errorf("%s failed, unexpected count of PVC annotations returned, Want: 0; Got: %d", tc.name, len(tc.pvc.Annotations))
+
+			assert.NotNil(t, tc.pvc.Annotations, "unexpected PVC annotations returned, Want: non-nil; Got: nil")
+
+			if tc.expectNonEmptyAnnotations {
+				assert.Emptyf(t, tc.pvc.Annotations, "unexpected count of PVC annotations returned, Want: 0; Got: %d", len(tc.pvc.Annotations))
```
Comment (skriss) [inline velero-csi-plugin/csi_restorer_test.go]: something seems reversed here - when tc.expect**Non**EmptyAnnotations is true, we assert that tc.pvc.Annotations is empty?

Comment (ashish-amarnath) [reply velero-csi-plugin/csi_restorer_test.go]: That was confusing. I updated the tests to make it more clear. PTAL.

File: velero-csi-plugin/csi_restorer_test.go [hunk 2]
```diff
@@ -90,33 +99,31 @@ func TestResetPVCAnnotations(t *testing.T) {
...
 			resetPVCAnnotations(&tc.pvc, tc.preserveAnnotations)
-			if tc.pvc.Annotations == nil {
-				t.Errorf("%s failed, unexpected PVC annotations returned, Want: non-nil; Got: nil", tc.name)
-			}
-			if tc.expectNonEmptyAnnotations && len(tc.pvc.Annotations) != 0 {
-				t.Errorf("%s failed, unexpected count of PVC annotations returned, Want: 0; Got: %d", tc.name, len(tc.pvc.Annotations))
-			}
-
-			if tc.expectNonEmptyAnnotations {
+			if tc.expectEmptyAnnotations {
```
Comment (skriss) [inline velero-csi-plugin/csi_restorer_test.go]: OK, so now that I'm looking at this more closely, I think we can remove L115-126 and replace it with `assert.Equal(t, tc.expectedAnnotations, tc.pvc.Annotations)`, right? 

If so, then can also remove the `expectEmptyAnnotations` field from the anonymous test case struct.

Comment (ashish-amarnath) [reply velero-csi-plugin/csi_restorer_test.go]: Lol! Yes. That would def simplify this!
Updated the tests. Thanks for catching this!

**Pull request merged**